include/fastdds/rtps/participant/ParticipantDiscoveryInfo.h

```diff
@@ -46,7 +46,7 @@ struct ParticipantDiscoveryInfo
         REMOVED_PARTICIPANT,
         DROPPED_PARTICIPANT,
         IGNORED_PARTICIPANT
-    };
+    }
```

Comment:
Can you revert this? Compilation failed
```suggestion
    };
```